src/plugins/query_enhancements/server/connections/managers/prometheus_manager.ts

```diff
@@ -97,6 +104,8 @@ class PrometheusManager extends BaseConnectionManager<OpenSearchClient> {
         return `${BASE_RESOURCE_API}/alerts`;
       case PROMETHEUS_RESOURCE_TYPES.LABEL_VALUES:
         return `${BASE_RESOURCE_API}/label/${resourceName}/values`;
+      case PROMETHEUS_RESOURCE_TYPES.METRICS:
+        return `${BASE_RESOURCE_API}/label/__name__/values`;
       case PROMETHEUS_RESOURCE_TYPES.METRIC_METADATA:
         const metricMetadataQueryString = resourceName ? `?metric=${resourceName}` : '';
         return `${BASE_RESOURCE_API}/metadata${metricMetadataQueryString}`;
```

Comment:
this can actually have browser call the API with query param `metric=${resourceName}` rather than constructing string here, it would allow query params other than `metric`, handles escaping etc.

Reply:
Make sense. Mind if I address this in followup? I can update this across this prometheus manager and the corresponding resource client.